tools/setup_permissions.sh

```diff
@@ -56,6 +65,12 @@ function grant_task_execution() {
          "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:master:task:user:root" \
          -d '{"description":"Execute Mesos tasks as user=root"}' \
          -H 'Content-Type: application/json'
+    # XXX see above
+    curl -k -L -X PUT \
+         -H "Authorization: token=${ACS_TOKEN}" \
+         "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:agent:task:user:root" \
```

Comment:
I believe this is wrong. We should only need to grant the SERVICE_USER, FRAMEWORK_USER, or LINUX_USER (all the same thing semantically, but named different things throughout). We should not need to grant to 'root' unless the scheduler or pods are running as 'root'.